bindata/assets/deployments/console-deployment.yaml

```diff
@@ -56,6 +75,7 @@ spec:
             - "--public-dir=/opt/bridge/static"
             - "--config=/var/console-config/console-config.yaml"
             - "--service-ca-file=/var/service-ca/service-ca.crt"
+            - "--downloads-server"
```

Comment:
This flags is to determine if goroute, which is responsible for running the handler, should run ?